crates/bevy_pbr/src/meshlet/gpu_scene.rs

```diff
@@ -781,13 +790,30 @@ impl FromWorld for MeshletGpuScene {
             ),
             downsample_depth_bind_group_layout: render_device.create_bind_group_layout(
                 "meshlet_downsample_depth_bind_group_layout",
-                &BindGroupLayoutEntries::sequential(
-                    ShaderStages::FRAGMENT,
+                &BindGroupLayoutEntries::sequential(ShaderStages::COMPUTE, {
+                    let write_only_r32float = || {
+                        texture_storage_2d(TextureFormat::R32Float, StorageTextureAccess::WriteOnly)
+                    };
                     (
-                        texture_2d(TextureSampleType::Float { filterable: false }),
+                        texture_depth_2d(),
+                        write_only_r32float(),
+                        write_only_r32float(),
+                        write_only_r32float(),
+                        write_only_r32float(),
+                        write_only_r32float(),
+                        texture_storage_2d(
+                            TextureFormat::R32Float,
+                            StorageTextureAccess::ReadWrite,
+                        ),
```

Comment:
Can you add a comment saying that this is mip 6? Having to count the mips is kinda annoying.